modules/ROOT/pages/multi-factor-authentication.adoc

```diff
@@ -5,10 +5,10 @@ endif::[]
 
 Multi-factor authentication (MFA) provides an additional layer of verification for Anypoint Platform users.
 MuleSoft requires MFA when users access Anypoint Platform, either directly with their Anypoint credentials or via single sign-on (SSO).
-MFA is enabled for direct logins and can’t be turned off at the organization level. However, there are a few service account types that don’t require MFA. An organization administrator should waive MFA for these accounts to allow existing integrations to continue without interruption.
+MFA is enabled for direct signins and can’t be turned off at the organization level. However, there are a few service account types that don’t require MFA. An organization administrator should waive MFA for these accounts to allow existing integrations to continue without interruption.
```

Comment:
```suggestion
MFA is enabled for direct signins and can’t be turned off at the organization level. However, a few service account types don’t require MFA. An organization administrator should waive MFA for these accounts so that existing integrations can continue without interruption.
```